.github/workflows/unit-tests-and-lint.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           ./scripts/fix-reports-path-in-github-runner.sh
 
       - name: SonarQube Scan
-        uses: SonarSource/sonarqube-scan-action@v5
+        uses: SonarSource/sonarqube-scan-action@0303d6b62e310685c0e34d0b9cde218036885c4d  # v5.0.0
```

Comment:
This is also whitelisted:
```
sonarsource/sonarqube-scan-action@* 
```

Therefore, we have the option to keep this action as it is.

Reply:
Yes, but still wanted to make sure that an exact version is for this external action.